pkgs/by-name/li/libsignal-ffi/package.nix

```diff
@@ -0,0 +1,55 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, runCommand, xcodebuild, protobuf, boringssl }:
+let
+  # boring-sys expects the static libraries in build/ instead of lib/
+  boringssl-wrapper = runCommand "boringssl-wrapper" { } ''
+    mkdir $out
+    cd $out
+    cp -r ${boringssl.out}/lib build
+    cp -r ${boringssl.dev}/include include
+  '';
```

Comment:
Could we also do this with `ln -s` instead of `cp -r`? This might save some disk space.

Reply:
Done, looks like boring-sys is fine with symlinks :)